openreview/venue/recruitment.py

```diff
@@ -55,6 +55,8 @@ def invite_committee(self,
         invitation_id = invitation.id
         hash_seed = invitation.content['hash_seed']['value']
 
+        invitees = [invitee.lower() if not invitee.startswith('~') else invitee for invitee in invitees]
```

Comment:
Can you check by "@" instead for tilde id, like the API 1:

```
invitees = [e.lower() if '@' in e else e for e in invitees if len(e) > 0]
```

Reply:
oh I hadn't noticed we were doing something similar for API1! Changed.